Task 4/Task 4/Task 4/Reader.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Task_4
+{
+    public static class Reader
```

Comment:
A bookworm, I see.

I think you should move methods from Reader and Writer to some `LogsSerializer` that would read and write this log file. Better if it is not static (to set log file name in ctor and just call methods)

ReadContent is the same as File.ReadAllText()